src/m1_facilitator/handlers/ConfirmWithdrawIntentsHandler.ts

```diff
@@ -62,7 +63,9 @@ export default class ConfirmWithdrawIntentsHandler extends ContractEntityHandler
   public async handle(records: ConfirmWithdrawIntentsEntityInterface[]): Promise<void> {
     Logger.info(`ConfirmWithdrawIntentsHandler::records received: ${records.length}`);
     const savePromises = records.map(async (record): Promise<void> => {
-      const gatewayRecord = await this.gatewayRepository.get(record.contractAddress);
+      const gatewayRecord = await this.gatewayRepository.get(
+        Utils.toChecksumAddress(record.contractAddress),
```

Comment:
It should be globalAddress also.
```suggestion
        Gateway.getGlobalAddress(Utils.toChecksumAddress(record.contractAddress)),
```